realm/realm-library/src/androidTest/kotlin/io/realm/KotlinSchemaTests.kt

```diff
@@ -49,7 +49,7 @@ class KotlinSchemaTests {
 
     @Test
     fun kotlinTypeNonNull() {
-        val objSchema = realm.schema.get(AllKotlinTypes::class.simpleName)!!
+        val objSchema = realm.schema.get(AllKotlinTypes::class.simpleName!!)!!
 
         // Document current nullability. Ideally all should be non-nullable. This is currently
```

Comment:
This whole test is pretty weird.  Is it testing only something that should not be true?  I looked at the related issue and couldn't find anything that seemed to pertain, so maybe  it would be cool to say a little more about what's going on in there...